src/main/java/io/wispforest/lavender/structure/BlockStatePredicate.java

```diff
@@ -54,13 +59,14 @@ public boolean isOf(MatchCategory type) {
         }
     };
 
-    Result test(BlockState state);
+    @NotNull
+    Result test(@NotNull BlockState state);
 
     /**
      * @return {@code true} if this predicate finds a {@linkplain Result#STATE_MATCH state match}
      * on the given state
      */
-    default boolean matches(BlockState state) {
+    default boolean matches(@NotNull BlockState state) {
```

Comment:
If not explicitly annotated as `@Nullable`, a parameter is assumed to be non-nullable and thus there's no need for this annotation especially seeing how almost nothing else in Lavender uses it either - be consistent

Reply:
alright

I added it as habit, but I'll remove all instances of it later